File: 정수빈/Lv2/JadenCase 문자열 만들기.java
import java.util.*;

class Solution {

  public String solution(String s) {
    String words[] = s.split(" ");
    String answer = "";

    for (int i = 0; i < words.length; i++) {
      String word = words[i];

      words[i] =
          word.substring(0, 1).toUpperCase() + word.substring(1, word.length()).toLowerCase();
    }

    for (int i = 0; i < words.length; i++) {
      String str = words[i];
      if (i == words.length - 1) {
        answer += str;
      } else {
        answer += str + " ";
      }
    }
    return answer;
  }
}

File: 정인희/LV2/JadenCase 문자열 만들기.py
def solution(s):
    answer = ''

    l = []
    l.append(s.split(' '))
    # l.append(list(map(list,s.split(' '))))
    # print(l)

    for i in s.split(' '):
        i = str(i)
        i = i.capitalize()
        print(i)
        answer += i
        answer += " "
    answer = answer[:-1]
    print(answer)
    return answer